lichess-bot.py

```diff
@@ -118,6 +118,12 @@ def game_error_handler(error):
     logger.exception("Game ended due to error:", exc_info=error)
 
 
+def should_restart(li, username):
+    online_bots = li.get_online_bots()
+    bot = list(filter(lambda bot: bot["username"] == username, online_bots))
+    return bot and bot[0].get("online")
```

Comment:
I don't understand this test. I can understand checking that my bot is in the returned list. But, when I look at the my bot's JSON, there's no `"online"` field, so the result is False because `get()` returns `None`. But, if the data had `online: true`, that would cause a restart?

Reply:
This PR does work for me, but I've never had trouble with losing the connection to lichess.

Reply:
I fixed the check at line 124. According to the [lichess API docs](https://lichess.org/api#operation/apiBotOnline) there should be an "online" field. I remember in the past [lichess](https://lichess.org/player/bots) would show also a few bots that were offline, so maybe this was the reason that they added the online field. Probably they have fixed it, so it now only returns bots that are online, so they don't send the "online" value.

I also never had a problem with my bot losing connection to lichess, but I only use my bot for testing. This is just an attempt to fix the problem that was reported in the PRs mentioned above.

Reply:
The second check with "online" was removed since the API stopped returning the "online" field and it now only returns bots that are online.